clusters/build-clusters/vsphere/ci-operator/admin_test-credentials.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
Whats the purpose of all of this?

Reply:
Probably we wont run our ci-tools tests there on `vsphere`.

https://github.com/openshift/release/blob/ac2a12b7e8e33d360cc8ed9319277c0a9a7a2f86/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L349-L354

Reply:
Removed.